tests/PHPStan/Rules/Methods/data/bug-5436.php

```diff
@@ -4,7 +4,6 @@
 
 final class PDO extends \PDO
 {
-	#[\ReturnTypeWillChange]
```

Comment:
deleted this line because it lead to different error lines beeing reported depending on php version/github action job